en/micro-integrator/docs/references/connectors/salesforce-rest-connector/sf-rest-inbound-endpoint-example.md

```diff
@@ -0,0 +1,123 @@
+# Salesforce Inbound Endpoint Example
+
+In the real world, the Salesforce streaming Inbound Endpoint allows you to perform various operations on Salesforce streaming data via WSO2 EI.
+
+The Salesforce streaming API receives notifications based on the changes that happen to Salesforce data with respect to an SQQL (Salesforce Object Query Language) query you define, in a secured and scalable way. For more information, navigate to [Salesforce streaming documentation](https://developer.salesforce.com/docs/atlas.en-us.202.0.api_streaming.meta/api_streaming/quick_start_workbench.htm).
+
+## What you'll build
+
+The Salesforce inbound endpoint supported via the WSO2 EI is a listening inbound endpoint that can consume messages from the Salesforce. Then injects messages to a sequence in WSO2 EI. However, for simplicity of this example, we will just log the message. You can extend the sample as required using WSO2 [EI mediators](https://docs.wso2.com/display/EI660/ESB+Mediators). 
```

Comment:
```suggestion
The Salesforce inbound endpoint supported via the WSO2 EI is a listening inbound endpoint that can consume messages from Salesforce. This injects messages to a sequence in WSO2 EI. However, for simplicity of this example, we will just log the message. You can extend the sample as required using WSO2 [EI mediators](https://docs.wso2.com/display/EI660/ESB+Mediators). 
```

Let's change this link to a relative link in the 7.0.0 docs.

Reply:
Updated.